calculator.js

```diff
@@ -14,78 +15,79 @@ button0.addEventListener("click", function() {
     const newValue = currentValueOfResult + 0;
     console.log(parseInt(newValue));
     result.textContent = newValue;
+    if (_firstOperand === "+" || "-" || "*" || "/") result.textContent = "";
 });
 
 button1.addEventListener("click", function() {
     const currentValueOfResult = result.textContent;
     const newValue = currentValueOfResult + 1;
     console.log(parseInt(newValue));
     result.textContent = newValue;
-
+    if (_firstOperand === "+" || "-" || "*" || "/") result.textContent = "";
```

Comment:
This check need to be done at the beginning of the function

Reply:
ok thanks